scripts/initialize_database.py

```diff
@@ -1,4 +1,6 @@
 # from donate.database import db
+from dotenv import load_dotenv
+load_dotenv()
```

Comment:
Based on the [python-dotenv readme](https://github.com/theskumar/python-dotenv#user-content-getting-started), it looks like the file that calls `dotenv.load_dotenv` needs to be in the same directory as the `.env` file which does not appear to be the case here.

Reply:
This shouldn't have been left in there, thanks.